eventarc/generic/Gemfile.lock

```diff
@@ -62,4 +62,4 @@ DEPENDENCIES
   sinatra (~> 2.0)
 
 BUNDLED WITH
-   2.1.4
+  1.17.3
```

Comment:
Similarly, we really should stay with bundler 2 to build the runtime image.

Same with the pubsub sample below.

Reply:
I've had to downgrade elsewhere too (https://github.com/GoogleCloudPlatform/ruby-docs-samples/pull/716) due to the test infra setup. Without this the tests fail and stop. The dockerfile needs to be downgrade to work with this bundler version from my understanding.

Reply:
Yes, our test infra right now is somewhat broken in that it doesn't have bundler 2 installed, so for _most_ tests, Gemfiles need to reference bundler 1. However, I think _these_ bundles are not installed by the test infra, but are part of the images that get deployed to cloud run (i.e. the bundle install happens inside the Dockerfile.) So I _think_ we should be able to use an up-to-date bundler and an up-to-date Ruby.

Reply:
I've reverted the changes, but see an error:
```
bundle exec ruby /tmpfs/src/github/ruby-docs-samples/spec/e2e_cleanup.rb eventarc/generic 1a33ed0017
/usr/local/lib/ruby/site_ruby/2.4.0/bundler/lockfile_parser.rb:108:in `warn_for_outdated_bundler_version': You must use Bundler 2 or greater with this lockfile. (Bundler::LockfileError)
```

Reply:
Ugh. Looks like e2e_cleanup is doing the wrong thing; for one thing, it's assuming that e2e means app engine (which it clearly doesn't in this case). And it shouldn't be trying to install the bundle in those directories—that's a useless exercise. I suspect a long-term fix is going to require much deeper surgery into the test scripts.

Maybe for now it's okay to switch the bundler and Ruby versions to deal with the immediate test failures, much as it makes me sad.

Reply:
I've reverted the changes. Thanks for looking at this PR. I will add these notes to the sample testing WG sync.

Reply:
Thanks Averi. Sorry about the churn on this. Graham and I will need to take a closer look at all this and get it cleaned up. The rest looks fine in the meantime, so LGTM.